README.md

```diff
@@ -314,6 +298,22 @@ The network is defined as the communication path over which a host or network ev
 | <a name="network.total.packets"></a>network.total.packets | Network outbound packets. The sum of inbound.packets + outbound.packets | core | long | `24` |
 
 
+## <a name="observer"></a> Observer fields
+
+An observer is defined as a special network, security, or application device used to detect, observe, or create network, security, or application-related events and metrics. This could be a custom hardware appliance or a server that has been configured to run special network, security, or application software. Examples include firewalls, intrusion detection/prevention systems, network monitoring sensors, web application firewalls, data loss prevention systems, and APM servers. The observer.* fields shall be populated with details of the system, if any, that detects, observes and/or creates a network, security, or application event or metric. Message queues and ETL components used in processing events or metrics are not considered observers in ECS.  
```

Comment:
@graphaelli Can you leave your comment on this one? APM-Server is in here :-)

Reply:
Elastic APM server typically operates as more as a message queue using this definition.  Would logstash be considered an observer?

Reply:
I would say even if APM server doesn't do a lot of work on the payload, compared to the library installed in each app, it could still be considered as the observer.

Reply:
Perhaps it makes sense for an observer simply to be an entity that works on behalf of an agent to report its data.

Reply:
@graphaelli I think our definition of observer needs to extend beyond what you suggest to include firewalls and network IDS, etc. that don't have an agent at all.

Reply:
@graphaelli Yes, LS is in many cases also the observer.

Reply:
I agree that compared to the agent the server doesn't do a lot with the payload currently. However, I am not sure that this is a general enough criteria for not considering the server an ETL component. There are some additions and changes the server does, and going forward with ECS, the server will change the structure of events even more in the future, as there need to be a mapping from the incoming events to the ES ECS structure. 

My point is, I assume that 

> Message queues and ETL components used in processing events or metrics are not considered observers in ECS.

should clarify what to consider an observer and what not, but for me it introduces more confusion.